server/jobs/submitManualJob.js

```diff
@@ -15,11 +15,15 @@ const dispatchAction = (action,data) =>  parentPort.postMessage({ action, data }
 
 (async () => {
 	logToConsole("Send notification ...");	
+	let jobExecution;
 	try {
-		const jobExecution = await JobExecution.create(workerData);
+		jobExecution = await JobExecution.create(workerData);
 		workerData.url = `${process.env.WEB_URL}${workerData.applicationId}/manualJobDetails/${workerData.jobId}/${jobExecution.id}`;
 		await assetUtil.notifyManualJob(workerData);
 	}catch (err) {
+		if(jobExecution){
+			jobExecution.update({status : 'error'})
```

Comment:
``	jobExecution.update({status: 'error'})`` is an async operation, it should have await in front of it , rather than that, ready to merge